full-moon/tests/roblox_cases/pass/types_generic_declaration/source.lua

```diff
@@ -0,0 +1,4 @@
+type Foo<T, U> = (T...) -> U...
```

Comment:
a minor suggestion, make these files for OSS Luau can correctly pass analysis on these files. at some point, we may want to include that cross-check in CI.
```suggestion
--!strict
type Foo<T, U> = (T...) -> U...
```